testing/machine/pod.go

```diff
@@ -71,6 +71,13 @@ func (m *Machine) WaitForPodsDelete(namespace string) error {
 	})
 }
 
+// WaitForPodRestart blocks until the pod is restarted. It fails after the timeout.
```

Comment:
Is `StartTime` set to nil after some time? Otherwise this would be true, every time a pod has been restarted once.
Which is ok for tests, but I wonder if the comment should say so.

Reply:
Actually, I was writing for bosh dns pods, but turns the it has random name and it get killed and new randon name pod is generated. So this PR is not necessary.